package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-import": "1.10.3",
     "eslint-plugin-jsx-a11y": "2.0.1",
     "eslint-plugin-react": "5.2.2",
-    "extract-text-webpack-plugin": "1.0.1",
+    "extract-text-webpack-plugin": "^2.0.0-beta.3",
```

Comment:
I think we must remove caret because it's in beta and api can be changed.
I faced with this today.